types/tablestore/helper.d.ts

```diff
@@ -0,0 +1,2 @@
+type EnumValues<T> = T[keyof T];
+type EnumValuesOrKeys<T> = T[keyof T] | keyof T;
```

Comment:
Sorry I missed this; these become globals for everyone. Let’s move them into `index.d.ts` (don’t need to be exported) so they’re locals.

Reply:
OK, thanks. I moved them, and added 'export {}' to shut off automatic exporting.